src/js/components/HeaderBar.tsx

```diff
@@ -1,20 +1,67 @@
 import * as React from "react";
+import AuthStore from "#SRC/js/stores/AuthStore";
+
 import { HeaderBar as UIHeaderBar } from "ui-kit-stage/HeaderBar";
+import AccountDropdown from "./AccountDropdown";
+
+function getUserAccountDropdown() {
+  const menuItems = [
+    {
+      html: "Logout",
+      id: "logout",
+      onClick: AuthStore.logout
+    }
+  ];
+
+  var userLabel = AuthStore.getUserLabel();
+  return (
+    <AccountDropdown
+      menuItems={menuItems}
+      userName={userLabel}
+      willAnchorRight={false}
+    />
+  );
+}
+
+function getClusterDropdown() {
+  const menuItems = [
+    {
+      html: "Install CLI",
+      id: "install-cli",
+      onClick: AuthStore.logout
+    }
+  ];
+
+  const userLabel = "frontend-us-west-2-bit1j110bd";
+
+  return (
+    <AccountDropdown
+      menuItems={menuItems}
+      userName={userLabel}
+      willAnchorRight={true}
+    />
+  );
+}
 
 export default function HeaderBar() {
   // remove this to activate component
   if (arguments) {
     return null;
   }
 
+  var userAccountDropdown = getUserAccountDropdown();
+  var clusterDropdown = getClusterDropdown();
+
   return (
     <UIHeaderBar>
       <span>toggle component here</span>
       <div className="header-bar-logo-wrapper">
         <div className="header-bar-logo" />
       </div>
-      <span>user-menu component here</span>
-      <span>cluster-menu component here</span>
+      <div className="header-bar-dropdown-wrapper">
```

Comment:
This PR is only about the user menu, would love to see the code not touching the cluster menu to prevent conflicts with @DanielMSchmidt :/ (@DanielMSchmidt does this work for your code?)

Reply:
Yes, I can remove the cluster dropdown part of this PR. I added because I needed to see how the user dropdown would look/perform adjacent to another dropdown (and so you guys could check as well).